accounts/l10n/.tx/config

```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://www.transifex.com
+
+[ocis.accounts]
```

Comment:
I'd prefer something like `owncloud.ocis-accounts`, because I'd like to stay in the `owncloud` project. Having `ocis.accounts` would require an entirely new transifex project and we would loose the benefit of reusing translations already existing from other resources.